pypeit/spectrographs/gemini_gmos.py

```diff
@@ -860,8 +946,8 @@ def get_detector_par(self, det, hdu=None):
             nonlinear       = 0.95,
             mincounts       = -1e10,
             numamplifiers   = 4,
-            gain            = np.atleast_1d([1.74]*4),  # Slow Readout,	Low Gain
-            ronoise         = np.atleast_1d([3.75]*4),  # Slow Readout,	Low Gain
+            gain            = np.atleast_1d([1.652, 1.761, 1.724, 1.813]),  # Slow Readout,	Low Gain
+            ronoise         = np.atleast_1d([3.460, 3.350, 3.250, 3.500]),  # Slow Readout,	Low Gain
```

Comment:
Nice!  We should make sure to comment on this update in the release notes.

Reply:
Done